care/facility/models/patient.py

```diff
@@ -683,3 +687,23 @@ class PatientNotes(FacilityBaseModel, PatientRelatedPermissionMixin):
         null=True,
     )
     note = models.TextField(default="", blank=True)
+
+    editable_until = models.DateTimeField(null=True, blank=True)
```

Comment:
`created_date` should work fine. If we decide to change the timeframe window, the edit window on previous Note objects on would also be changed in case of using `created_date`. That should be the preferred behaviour right?

Reply:
I think it's okay, because we won't be changing it drastically.

Reply:
Will remove `editable_until`.